packages/shared/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { existsSync, readFileSync } from 'fs';
+
+export const getMarkdownContents = (filePath: string, fallbackContent: string): string => {
```

Comment:
since this is a shared methos,  tell from the function content itself, not necessarily  MarkDown contents?

Reply:
changed to `getFileContents`